### PR TITLE
Update to NixOS 24.11

### PR DESCRIPTION
**Pull request by mweinelt**
- [x] umbriel
- [x] caliban

### CODE REVIEW

File: non-critical-infra/modules/vaultwarden.nix [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   services.vaultwarden = {
     enable = true;
-    backupDir = "/var/lib/bitwarden_rs/backups/";
+    backupDir = "/var/backup/vaultwarden/";
```
Comment (jfly) [inline non-critical-infra/modules/vaultwarden.nix:10]: Out of curiosity, why the change?

**Review by jfly — APPROVED** [review]

File: non-critical-infra/modules/vaultwarden.nix (hunk 1 above)
Comment (mweinelt) [reply non-critical-infra/modules/vaultwarden.nix:10]:
```
       error:
       Failed assertions:
       - Backup directory can not be in /var/lib/bitwarden_rs

**Comment by jfly** [comment]
How does this PR interact with https://github.com/NixOS/infra/pull/517?

**Comment by mweinelt** [comment]
It won't eval because of the failed assertion.

**Review by Mic92 — APPROVED** [review]
